app/models/user.rb

```diff
@@ -1,12 +1,37 @@
 class User < ApplicationRecord
-  before_save{self.email = email.downcase!}
+  attr_accessor :remember_token
+  before_save{self.email = email.downcase}
   validates :name, presence: true,
-    length: {maximum: Settings.user.name.length.maximum}
+  length: {maximum: Settings.user.name.length.maximum}
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
   validates :email, presence: true,
-    length: {maximum: Settings.user.email.length.maximum},
-    format: {with: VALID_EMAIL_REGEX}, uniqueness: {case_sensitive: false}
+  length: {maximum: Settings.user.email.length.maximum},
+  format: {with: VALID_EMAIL_REGEX}, uniqueness: {case_sensitive: false}
   has_secure_password
   validates :password, presence: true,
-    length: {minimum: Settings.user.password.length.minimum}
+  length: {minimum: Settings.user.password.length.minimum}, allow_nil: true
```

Comment:
` allow_nil: true` có ý nghĩa gì em

Reply:
allow_nil: true cho phép bỏ qua validation khi giá trị là nil

Reply:
Chưa đủ em ạ, đăng ký user chị ko nhập trường passwork nó lại báo là paswork can not blank

Reply:
rep lại nôi dung nàyem